### PR TITLE
Allow a port to be opened for CDP proxy

### PR DESCRIPTION
**Pull request by arjunrn**
The CDP Proxy service needs SSL to be terminated on the application.

Signed-off-by: Arjun Naik <arjun.rn@gmail.com>

### CODE REVIEW

File: cluster/cluster.yaml [hunk 1]
```diff
@@ -272,6 +272,13 @@ Resources:
           FromPort: 30443
           IpProtocol: tcp
           ToPort: 30444
+{{- end }}
+{{- if index .Cluster.ConfigItems "open_cdp_proxy_port" }}
```
Comment (aermakov-zalando) [inline cluster/cluster.yaml]: At this point I'd switch to something like `open_worker_ports: "30443-30444,30445-30445"` and then use templating to generate it.

**Comment by szuecs** [comment]
Do we want to go forward and clean up later or do you want to wait until the change you made in clm is life? @arjunrn

**Comment by arjunrn** [comment]
I will wait till the CLM change is deoloyed

**Comment by arjunrn** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**